lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Settings.hs

```diff
@@ -66,8 +70,8 @@ spec = describe "SHELLEY_SETTINGS" $ do
     it "SETTINGS_02 - Changing pool_metadata_source re-syncs metadata" $ \ctx -> do
         let toNone = "none"
             toDirect = "direct"
-            getMetadata = fmap (view #metadata) . snd <$> unsafeRequest
-                @[ApiStakePool] ctx (Link.listStakePools arbitraryStake) Empty
+            getMetadata = fmap (^. _Unwrapped . #metadata) . snd <$> unsafeRequest
```

Comment:
```suggestion
            getMetadata = fmap (getApiT . view #metadata) . snd <$> unsafeRequest
```
? 🤔

As much as I feel the pain of unwrapping, `_Unwrapped` generates significant cognitive overload in that a) it pulls in `Data.Generics` and b) it's not clear to which type it is applied, and may hence to do the wrong thing.

Reply:
Agree